config/environments/production.rb

```diff
@@ -45,7 +45,7 @@
   # config.action_cable.allowed_request_origins = [ 'http://example.com', /http:\/\/example.*/ ]
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
```

Comment:
Let's also add

```ruby
  config.ssl_options = { redirect: false, hsts: false } 
```

We handle the redirect/HTST elsewhere in our stack, and this will let us continue to boot the app in production mode locally for testing.

Reply:
Done